src/viam/examples/camera/example_camera.cpp

```diff
@@ -1,6 +1,5 @@
 #include <fstream>
 #include <string>
-#include <unistd.h>
```

Comment:
I don't know why all these files included `<unistd.h>` and at least on my mac they appear to compile without it. We will see what CI says.